sidh/sidh_test.go

```diff
@@ -1,4 +1,4 @@
-package p751sidh
+package sidh
```

Comment:
It's actually fine to have the p751toolbox used in the testing of the sidh package, even if the sidh package is stripped down to just the function definitions.

Reply:
Noted. I will incorporate it